generator/index.js

```diff
@@ -189,6 +248,14 @@ async function parse() {
         }
     }
 
+    // Normalize the new "Smileys & Emotion" and "People & Body" categories to the ones we have.
+    const smileysAndEmotion = result.get("SmileysAndEmotion")
+    const peopleAndBody = result.get("PeopleAndBody")
+
+    result.set("SmileysAndPeople", smileysAndEmotion.concat(peopleAndBody))
+    result.delete("SmileysAndEmotion")
+    result.delete("PeopleAndBody")
```

Comment:
Our upstream source slightly changed the categories. Here we normalize them back to what we had previously.